.gitmessage

```diff
@@ -0,0 +1,13 @@
+[PFML-(0-9)*] - Feature Summary
```

Comment:
nit: Adding the PFML part doesn't seem needed? It takes up characters for the short summary, the link is provided in the resources section, and we won't always be able to keep the convention (since not every commit will have an associated jira link, or if we choose to change from jira).

Reply:
I put the PFML there to easily see what jira we were working on. I can remove it. No jira would just be [NO-JIRA]